stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/state/messages/list/MessageOptionItemState.kt

```diff
@@ -14,7 +14,7 @@ import io.getstream.chat.android.common.state.MessageAction
  * @param iconColor The color of the icon.
  * @param action The [MessageAction] the option represents.
```

Comment:
Not sure about the package here.

The state is located here:
`io.getstream.chat.android.compose.state.messages.list`

And the Composable itself in:
`io.getstream.chat.android.compose.ui.components.messageoptions`